### DISCUSSION TITLE
对空六课星见雅入驻（承接各类以骸清除业务）

### DISCUSSION THREAD

**Discussion by HoshimiMiyabi120** [share121/inter-knot#5839] [公告]
![Screenshot_20241226_203723](https://github.com/user-attachments/assets/ffd7d970-dd6f-4bca-aaf0-9f3a91caac05)
### 

> _**H·S·O·S·6提醒您:**_

绳网不是法外之地
拒绝虚狩饭圈文化，从每一个新艾利都市民做起

**Comment by Kiana175** [comment]
星见雅小姐有时间来我们崩坏世界坐坐呀！

**Comment by laoyu321** [comment]
星见雅小姐有时间来我们提瓦特大陆坐坐呀！[https://ys.mihoyo.com/](url)
